src/main/java/com/direwolf20/buildinggadgets/common/tools/NBTTool.java

```diff
@@ -31,15 +32,15 @@ public static NBTTagByteArray createBooleanList(Boolean[] booleans) {
 
     public static NBTTagList createShortList(short[] shorts) {
         NBTTagList list = new NBTTagList();
-        for (short s: shorts) {
+        for (short s : shorts) {
```

Comment:
Please don't do formatting changes in classes you don't even touch... They are just annoying when reviewing... (this is not a cleanup PR...)